frontend/src/components/dashboard/visitors/PeriodDropdown.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+    <VCol
+        class="d-flex"
+        cols="12"
+        sm="6"
+    >
+        <VSelect
+            :items="items"
+            item-text="title"
+            item-value="value"
+            class="option-select"
+            v-model="selectedItem"
+            solo
+            return-object
+            @change="change"
+        />
+    </VCol>
+</template>
+
+<script>
+    import {CHANGE_SELECTED_PERIOD} from "@/store/modules/visitors/types/actions";
+    import {mapActions} from 'vuex';
+
+    export default {
+        name: "PeriodDropdown",
+        data() {
+            return {
+                selectedItem: {
+                    value: 'last_week'
+                },
+                items: [
+                    {title: 'Today', value: 'today'},
+                    {title: 'Yesterday', value: 'yesterday'},
+                    {title: 'Last 7 days', value: 'last_week'},
+                    {title: 'Last month', value: 'last_month'},
+                    {title: 'Last quartal', value: 'last_quartal'},
+                    {title: 'All period', value: 'all_period'}
+                ]
+            };
+        },
+        methods: {
+            ...mapActions('visitors', {
+                changeSelectedPeriod: CHANGE_SELECTED_PERIOD
```

Comment:
not quite sure that you need Vuex store for this component, in general Vuex store shouldn't be used everywhere and in any case. In some cases consider to use local state with common components

For instance, it does make sense to specify selected value as prop and use v-model to track value change in parent component. Also take in account the if you using Vuex for every state management you Vuex store quite quickly become unsupported massive object. And also - Vuex will persist data, so each time when user will get back to this particular component this state will be restored. In addition to that - using Vuex for every key means that you will need to track and clear a lot of stuff on logout